src/routes/album.js

```diff
@@ -0,0 +1,17 @@
+// src/routes/album.js
+const express = require('express');
+const albumController = require('../controllers/album');
+
+const albumRouter = express.Router();
+
+albumRouter.post('/artist/:id/album', albumController.create);
+
+albumRouter.get('/album', albumController.read);
+
+albumRouter.get('/album/:id', albumController.readOne);
+
+albumRouter.patch('/album/:id', albumController.update);
+
+albumRouter.delete('/album/:id', albumController.delete);
```

Comment:
Could bunch these together
```suggestion
albumRouter.post('/artist/:id/album', albumController.create);
albumRouter.get('/album', albumController.read);
albumRouter.get('/album/:id', albumController.readOne);
albumRouter.patch('/album/:id', albumController.update);
albumRouter.delete('/album/:id', albumController.delete);
```